feed2discord.py

```diff
@@ -194,7 +195,8 @@ def sql_maintenance(config):
     # re-evaluating them.
     conn.execute(SQL_CLEAN_OLD_ITEMS)
 
-    conn.close()
+    if conn == None:
+        conn.close()
```

Comment:
The conn.close logic seems backwards? But maybe it's really that the conn param should have a different name than the `conn =` does, so that you can only close the conn if it made its own connection?